client-src/src/Facility/FacilityDetails.js

```diff
@@ -33,16 +35,38 @@ class FacilityDetails extends React.Component<Props> {
   }
 
   _handleArchive = async () => {
-    await this.props.postFormData(
-      { archived_date: new Date() },
-      "Facilities",
-      "PATCH",
-      "PATCH_BASIC_DETAILS",
-      "",
-      this.props.match.params.id
-    )
-    if (this.props.response) this.setState({ redirect: true })
-    else alert('Something went wrong')
+    confirmAlert({
+      customUI: ({ onClose }) => {
+        return (
+          <Col m={6} s={12} style={{ minWidth: '400px' }}>
+            <Card
+              title='Confirm'
+              className='blu darken-4'
+              textClassName='white-tex'
+              actions={
+                [
+                  <Button onClick={onClose} className="mfl-rm-2 btn-flat">No</Button>,
+                  <Button className="btn-flat" onClick={async () => {
```

Comment:
these should go into a function and you should just render them here to reduce noise